test/e2e/load_test.go

```diff
@@ -139,6 +139,9 @@ var _ = Describe("Podman load", func() {
 	})
 
 	It("podman load multiple tags", func() {
+		if podmanTest.Host.Arch == "ppc64le" {
+			Skip("skip on ppc64le")
```

Comment:
Is there a reason here?

Reply:
yeah, it has to do with what happens when two images are pulled with different tags and one doesn't really exist for the arch :)